packages/gtm-enhanced/src/__tests__/gtm-enhanced.test.ts

```diff
@@ -109,6 +109,46 @@ describe("GTM Enhanced", () => {
       ]);
     });
 
+    describe("#productAdded", () => {
+      it("maps to Enhanced Ecommerce spec", () => {
+        const anonId = analytics.user().anonymousId();
+
+        analytics.track("Product Added", {
+          name: "Monopoly: 3rd jEdition",
+          sku: "G-32",
+          price: 18.99,
+          category: "Games",
+          brand: "Hasbro",
+          variant: "200 pieces",
+          quantity: 1,
+          coupon: "MAYDEALS",
+          position: 3,
+          url: "https://www.example.com/product/path",
+          image_url: "https://www.example.com/product/path.jpg",
+        });
+
+        expect(window["dataLayer"]).toEqual([
+          expect.objectContaining({
```

Comment:
I believe it should look like this (source: https://developers.google.com/tag-manager/enhanced-ecommerce#add)